### PR TITLE
change back to Automation Analytics name

### PR DESCRIPTION
**Pull request by jlmitch5**
change back to Automation Analytics name
Screen Shot 2022-04-08 at 2 18 46 PM
Screen Shot 2022-04-08 at 2 12 46 PM

### CODE REVIEW

File: awx/ui/src/screens/Setting/Subscription/SubscriptionEdit/AnalyticsStep.js [hunk 1]
```diff
@@ -31,10 +31,10 @@ function AnalyticsStep() {
       spaceItems={{ default: 'spaceItemsMd' }}
       direction={{ default: 'column' }}
     >
-      <Trans>User and Insights analytics</Trans>
+      <Trans>User and Automation Analytics</Trans>
       <p>
         <Trans>
-          By default, we collect and transmit analytics data on the serice usage
+          By default, we collect and transmit analytics data on the service usage
```
Comment (nixocio) [inline awx/ui/src/screens/Setting/Subscription/SubscriptionEdit/AnalyticsStep.js]: 👍

**Review by nixocio — APPROVED** [review]
I just found references for `Insights for Ansible` on .po files.

**Comment by nixocio** [comment]
hey @jlmitch5 changes look good, but it seems like we need to update a couple of unit-tests.

**Comment by jlmitch5** [comment]
ThankI @nixocio I'll update the unit tests monday

**Review by InnocentK — APPROVED** [review]
Looks good! The above failing tests are unrelated to the changes in this PR so it should be fine to merge

**Pull request merged**